src/css/components/table.css

```diff
@@ -0,0 +1,121 @@
+/**
+ * @file
+ * Styles for Olivero's tables.
+ */
+ 
+table {
+  width: fit-content;
+  max-width: 100%;
+  border-collapse: collapse;
+  display: block;
+  margin: var(--sp2) 0;
+  border-spacing: 0;
+  border: 0;
+  font-family: var(--font-sans);
+  line-height: var(--sp1-5);
+  font-size: 16px;
+  color: var(--color--gray-10);
+  overflow-x: auto;
+  -webkit-overflow-scrolling: touch;
+  
+  background:
+        /* Left start and right start 'inside' container colors (they overlap the shadows) */
+				linear-gradient(90deg, white 0%, rgba(255,255,255,0)),
+        linear-gradient(-90deg, white 0%, rgba(255,255,255,0)) 100% 0,
+        /* Left and right scroll shadows */
+				radial-gradient(
+					farthest-side at 0% 50%,
+					rgba(0,0,0,0.2),
+					rgba(0,0,0,0)
+				),
+				radial-gradient(
+					farthest-side at 100% 50%,
+					rgba(0,0,0,0.2),
+					rgba(0,0,0,0)
+				) 100% 0%;
+			background-repeat: no-repeat;
+			background-color: white;
+			background-size: 100px 100%, 100px 100%, 14px 100%, 14px 100%;
+			background-attachment: local, local, scroll, scroll;
+
+  thead {
+    tr {
+      th {
+        border-bottom: 2px solid var(--color--blue-50);
+      }
+    }
+  }
+
+  tr {
+    white-space: nowrap;
+    background: transparent;
+
+    td,
+    th {
+      padding: var(--sp1-5) var(--sp1-5) var(--sp1-5) 0;
+    }
+
+    th {
+      background: white;
+      margin: 0;
+      letter-spacing: 0.12em;
+      text-transform: uppercase;
+      line-height: var(--sp1-5);
+      font-family: var(--font-sans);
+      font-size: 14px;
+      color: var(--color--gray-0);
+      text-align: left;
+
+      a {
+        color: var(--color--blue-20);
+        text-decoration: underline;
+        text-decoration-color: var(--color--blue-20);
+        text-decoration-width: 2px;
+        text-decoration-thickness: 2px;
+
+        @supports (box-shadow: none) {
+          text-decoration: none;
+          box-shadow: inset 0 -2px 0 0 var(--color--blue-50);
+          transition: box-shadow .3s cubic-bezier(.55,.085,0,.99);
+
+          &:hover,
+          &:focus {
+            box-shadow: inset 0 -30px 0 0 #d9ecfa;
+            color: black;
+            text-decoration: underline;
+            text-decoration-color: #d9ecfa;
+          }
+        }
+      }
+    }
+
+    td {
+      white-space: normal;
+      vertical-align: top;
+      min-width: 250px;
+      border-bottom: 2px solid var(--color--gray-40);
+    }
+  }
+
+  tbody {
+    tr {
+      th {
+        vertical-align: top;
+      }
+
+      &:last-child {
+        td {
+          border-bottom: 0;
+        }
+      }
+    }
+  }
+}
+
+/* When a table is placed in the body/text content area */
+.content-bleed {
+
+  @media (--xl) {
+    width: calc(10 * var(--grid-col-width--lg) + 4 * var(--grid-gap--lg));
+  }
```

Comment:
You're going to need to add CSS at all breakpoints similar to what I commented prior. I can help if needed.

Reply:
The screenshot that you've noted above doesn't seem to align with what's in the Figma (see attached). The table wrapper container is around `900px`, which is in the 10 column count. Let me know if I'm not seeing what I'm supposed to be seeing.

![D9_Theme_Master_–_Figma](https://user-images.githubusercontent.com/1815714/68898924-7acdf800-06fe-11ea-99f3-522da6685781.jpg)

![Screen Shot 2019-11-14 at 4 52 04 PM](https://user-images.githubusercontent.com/1815714/68899204-22e3c100-06ff-11ea-8603-88cc723d6608.png)